src/index.html

```diff
@@ -6,9 +6,56 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a class="header__logolink" href="#home">
+        <img src="images/logo.png" alt="logo">
+      </a>
+      <nav class="nav header__nav">
```

Comment:
an indent between multiline siblings
```suggestion
      </a>
      
      <nav class="nav header__nav">
```